libraries/fixlatest/src/main/java/com/paritytrading/philadelphia/fixlatest/FIXLatestEnumerations.java

```diff
@@ -7779,21 +7741,6 @@ private RiskLimitRequestResultValues() {
 
     }
 
-    /**
-     * Values for RiskLimitRequestStatus(1762).
-     */
-    public static class RiskLimitRequestStatusValues {
-
-        public static final int Accepted            = 0;
-        public static final int AcceptedWithChanges = 1;
-        public static final int Rejected            = 2;
-        public static final int AcceptancePending   = 3;
-
-        private RiskLimitRequestStatusValues() {
-        }
-
-    }
```

Comment:
RiskLimitRequestStatus(1762), among others, now shares its value range with PartyDetailRequestStatus(1878). It would be good to document these kind of dependencies for all dialects based on FIX Orchestra.